docs/getting-started/tutorials/auth-architecture/external-idp-internal-session.md

```diff
@@ -0,0 +1,210 @@
+---
+id: external-idp-internal-session 
+title: Use an external Identity Provider for the login phase but manage session and user groups within your project
+sidebar_label: External IdP + Internal session
+---
+
+This scenario is a slight modification of the previous-one: you keep your external Identity Provider (IDP) but the user will be linked to a custom session, and you can optionally manage the user groups within your project using a dedicated CRUD collection.
```

Comment:
This scenario is really complex and only useful if you want to handle yourself the token generation and the data of the user. I will add a disclaimer or introduction.

Reply:
An infobox has been added at the top of the page alerting the user to the difficulty and benefits of this scenario.